app/web/typo3conf/ext/theme/Classes/ViewHelpers/ScssBreakpointViewHelper.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Sup7even\Theme\ViewHelpers;
+
+use \TYPO3\CMS\Fluid\Core\ViewHelper\AbstractViewHelper;
+use \TYPO3\CMS\Extbase\Utility\DebuggerUtility;
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\Page\PageRenderer;
+use function GuzzleHttp\json_encode;
+
+/**
+ * parses a given scss file and extracts the breakpoint variables
+ */
+class ScssBreakpointViewHelper extends AbstractViewHelper {
```

Comment:
yeah. thats right. if we dont deploy scss files, we may write a little config file (json or whatevertype) for that purpose, maybe we can create that file dynamically in that viewhelper. if that file does not exist, create a "breakpoints.json" file for production mode eg.